django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,63 @@
+from django.conf import settings
+from django.contrib.auth.models import User as UserType
+from django.core.exceptions import PermissionDenied
+from django.db import models
+from ulid2 import generate_ulid_as_uuid
+
+from thunderstore.community.models import PackageListing
+from thunderstore.core.mixins import TimestampMixin
+from thunderstore.core.utils import capture_exception
+from thunderstore.repository.models.thread import CommentsThreadMixin, Thread
```

Comment:
Having the `models.thread` models be part of `models/__init__.py` caused cycling imports, and if I moved it to the top, I would get type errors. There's now a mix of `from thunderstore.repository.models.thread import <thread models>` and `from thunderstore.repository.models import <thread models>` as we don't define `__all__` in the models files or import specific names in `__init__.py`